### DISCUSSION TITLE
Rejected because of billing information

### DISCUSSION THREAD

**Discussion by beliboba** [community/community#140188] [GitHub Education]
### Hi everyone,

My request got rejected because my billing information doesnt contain my real name.
1. Why should i tell you  my real name and address if i do not plan to purchase anything
2. Even if i wanted to change the information i coulndt because of this error
```
Sorry, you can't update your billing information at this time
We are sorry, but after reviewing we have determined that the billing information supplied is ineligible for transactions with GitHub due to government restrictions.
[Learn more](https://docs.github.com/articles/github-and-trade-controls) about GitHub and trade control regulations.
```

**Comment by Tangc03** [comment]
I met the same problem and I'm wondering how to solve, have you found a solution now?

**Reply by beliboba** [reply]
no, github support avoids answering the question when asked directly

**Reply by Tangc03** [reply]
Yeah, I've met that as well. Anyway, thanks for your reply!

**Comment by horsizz** [comment]
Any solutions?

**Reply by YuuSanLearnCode** [reply]
https://github.com/orgs/community/discussions/70242

it fix for me
